add confidence testing script and batch API endpoint for top 5 dishes

Ahtishamu opened the pull request:
yeah

Ahtishamu committed:
add confidence testing script and batch API endpoint for top 5 dishes

HammadMal reviewed (changes requested):
Please update documentation for backend.

Ahtishamu committed:
Restore popular dish generation files

Ahtishamu commented:
> Please update documentation for backend

info about how to run is there.

Documentation for backend changes not needed, just changed things. merge it

Ahtishamu committed:
Update food detection documentation with batch endpoint and correct dish list

Ahtishamu commented:
> > Please update documentation for backend
> 
> info about how to run is there.
> 
> Documentation for backend changes not needed, just changed things. merge it

updated documentation
